app/controllers/api/v1/responses_controller.rb

```diff
@@ -71,7 +71,9 @@ def response_params
       end
 
       def add_chosen_emotion
-        { emotion: @response.emotion }
+        { emotion: @response.emotion,
+          user_shoutouts: current_user.shoutouts
+        }
```

Comment:
@VitaliiShevchenko the name of this method does not match its content and please change styling for hash to:
```
{ 
   emotion: @response.emotion,
   user_shoutouts: current_user.shoutouts
}
```

Reply:
fixed